testing/pow_test.py

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Minor nit, but I don't think we need to create a whole new file for just this one `pow` test.  Put it in `testing/op_test.py`, maybe?